code/datums/shuttles.dm

```diff
@@ -116,24 +116,24 @@
 
 /datum/map_template/shuttle/emergency/raven
 	suffix = "raven"
-	name = "CentCom Raven Cruiser"
-	description = "The CentCom Raven Cruiser is a former high-risk salvage vessel, now repurposed into an emergency escape shuttle. \
+	name = "NSV Raven"
+	description = "The NTSV Raven is a former high-risk salvage frigate, now repurposed into an emergency escape shuttle. \
```

Comment:
These names seem to disagree with each other -- should it be NSV or NTSV?

Reply:
whoops